apps/system/js/task_manager.js

```diff
@@ -745,7 +746,25 @@
           this.removeCard(card);
         }
         break;
+      case 'card-crossslidestart':
+        this.handleCardCrossSlide(evt);
+        break;
+    }
+  };
+
+  TaskManager.prototype.handleCardCrossSlide = function(evt) {
+    if (this._handlingCrossSlide) {
+      return;
     }
+    this._handlingCrossSlide = true;
+    var finish = (e) => {
+      if (this.element) {
+        this.element.style.overflowX = '';
+      }
+      this._handlingCrossSlide = false;
+    };
+    this.element.style.overflowX = 'hidden';
+    eventSafety(window, 'card-crossslideend', finish, 1000);
```

Comment:
let's work through the edge case and handle the `card-crossslideend` properly (ie. like the start event and without event safety).
it's pretty easy to shout ourselves in the foot with this version